application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -109,4 +109,13 @@ public static String abbreviate(String text, int maxLength) {
         return text.substring(0, maxLength - ABBREVIATION.length()) + ABBREVIATION;
     }
 
+    /**
+     * Mentions a guild channel by its id
+     *
+     * @param channelId the channel id to mention
+     * @return channel mention
+     */
+    public static String mentionChannelById(long channelId) {
+        return "<#%s>".formatted(channelId);
```

Comment:
%d is used for numbers

Reply:
i mean %s can take anything, but yea will do that